napari/layers/labels/_tests/test_labels.py

```diff
@@ -387,8 +388,8 @@ def test_custom_color_dict():
     )
 
     # test with custom color dict
-    assert type(layer.get_color(2)) == np.ndarray
-    assert type(layer.get_color(1)) == np.ndarray
+    assert isinstance(layer.get_color(2), ColorArray)
+    assert isinstance(layer.get_color(1), ColorArray)
```

Comment:
Given the changes in this PR, we can also do the following right?

```suggestion
    assert isinstance(layer.get_color(2), np.ndarray)
    assert isinstance(layer.get_color(1), np.ndarray)
```

That seems like the best approach. Other than maybe just deleting these assertions, which don't add much value to me.

Reply:
Yes, we can, but why not `ColorArray`?

Reply:
Don't mind what you do here. My reasoning is that `np.ndarray` is much more recognizable than `ColorArray`. I guess the coercion behavior means we have some more facts about what an instance of `ColorArray` means.

But like I said, I would be most in favor of just removing these assertions - they don't assert anything particularly interesting or relevant.